ai_diffusion/styles/default.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
accidental commit

Reply:
File removed. Should we ignore the styles directory?